ambry-replication/src/main/java/com.github.ambry.replication/ReplicaThread.java

```diff
@@ -780,6 +780,10 @@ private void writeMessagesToLocalStoreAndAdvanceTokens(List<ExchangeMetadataResp
             replicationMetrics.blobDeletedOnGetCount.inc();
             logger.trace("One of the blobs to GET is deleted: Remote node: {} Thread name: {} Remote replica: {}",
                 remoteNode, threadName, remoteReplicaInfo.getReplicaId());
+          } else if (partitionResponseInfo.getErrorCode() == ServerErrorCode.Blob_Authorization_Failure) {
+            replicationMetrics.blobAuthorizationFailureCount.inc();
+            logger.error("One of the blobs authorization failed: Remote node: {} Thread name: {} Remote replica: {}",
```

Comment:
minor: can you also print the keys that were part of the get (should be the `missingKeys`)